Write initial, simple SNI blocking experiment

bassosimone opened the pull request:
Same as other PRs I've opened recently: basic code. We'll wire in
the data-formats when we'll add code to process netx events.

We'll refine and declare stable only with 100% coverage including
making sure we're fully compliant with the spec.

See https://github.com/ooni/spec/pull/159 for the spec.

hellais commented on experiment/sniblocking/sniblocking.go:
```diff
@@ -0,0 +1,138 @@
+		tk.Behavior = "suspicious"
+		return
+	}
+	tk.Behavior = "🤷"
```
As mentioned in the ooni/spec repo I suggest using simple ascii for this as it will make parsing the data in all languages easier.

bassosimone commented:
@hellais I've addressed all your feedback, please take another look!

Regarding:

> Beyond that the test looks reasonable to me, I am curious to learn how this is going to be triggered by other tests though as my understanding is that the input to this experiment is some website which is already somehow suspicious.

Yes, this is what I was planning on doing.

bassosimone commented:
This is implemented as an automatic follow-up experiment in github.com/ooni/netx
